### ISSUE TITLE
Better dialog boxes

### ISSUE THREAD

**Issue by manparvesh** [IdeaTrackerPlus/IdeaTrackerPlus#12]
Description
-------------------------------------------
The current dialog boxes are too small and look congested

**Comment by JubrilEdu** [comment]
Hello, I'd love to work on this, can I get a little more information before proceeding?

**Comment by manparvesh** [comment]
@djubreel thanks for your interest!  
This is a UI related issue. When you tap on an idea, a dialog appears. I think that it would look better if the buttons are bigger and are visibly buttons, not look like links.

**Comment by JubrilEdu** [comment]
@manparvesh , I'd start work on it.

**Comment by JubrilEdu** [comment]
<img width="407" alt="screen shot 2017-10-07 at 2 13 53 pm" src="https://user-images.githubusercontent.com/5956953/31308185-ee526c90-ab69-11e7-8cb7-2b3d301ce8dd.png">
@manparvesh  is this fine?

**Comment by naushad-madakiya** [comment]
@djubreel Yes, looks good but make sure all the dialogs have the similar design, implementation of the color preference and don't break any existing functionality 👍

**Comment by JubrilEdu** [comment]
@naushad-madakiya according to @manparvesh I just needed to work on just this dialog which I have done.it follows the design of existing dialogs except for the button which was requested to be made to look more like a button.
